src/frontend/src/views/Privacy/index.tsx

```diff
@@ -27,7 +27,7 @@ const PrivacyPolicy = (): JSX.Element => {
   const renderIntro = () => (
     <>
       <Title>
-        <H1>Chan Zuckerberg GEN EPI (formerly Aspen) Privacy Policy</H1>
+        <H1>Chan Zuckerberg GEN EPI (formerly CZ Gen Epi) Privacy Policy</H1>
```

Comment:
This is not my call, but seems like a style mismatch.
Long name: "Chan Zuckerberg GEN EPI"
Short name: "CZ Gen Epi"

One of them has all caps for `gen epi`, while other uses title case? Is this actually what we want? @Janeece